packages/eui-usage-analytics/package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "eui-usage-analytics",
```

```suggestion
  "name": "@elastic/eui-usage-analytics",
  "private": true,
```

We usually prefer to keep all packages scoped to the default `@elastic` org, which comes with slightly more protection against accidental publication. Adding `private: true` won't hurt, too :)